src/sync.js

```diff
@@ -10,10 +10,10 @@ export function _sync(opts) {
 		unsub(client) {
 			clients = clients.filter(c => c != client);
 		},
-		pub(type, self, x, y, w, h, i) {
+		pub(type, self, x, y, w, h, i, ...extras) {
```

Comment:
please don't make this generic. spreads don't de-sugar well and the iife is compiled down to es5 which creates `arguments` bloat and creates extra intermediate arrays, just pass anything extra you need explicitly as i do in the other branch.

![image](https://user-images.githubusercontent.com/43234/82836775-db4ba800-9e8c-11ea-923f-5ebd187b2e28.png)